docs/core/versions/index.md

```diff
@@ -13,89 +13,220 @@ ms.assetid: f6f684b1-1d2c-4105-8376-7c1959e23803
-- APIs will only be added to NETStandard.Library packages as part of major or minor .NET Core releases, since doing so would require adding a new `netstandard` version. This is in addition to SemVer requirements.
+`NETStandard.Library` packages will typically version as a set, since they have implementation-level dependencies between them.
 
 ### Metapackages
 
-Versioning for .NET Core metapackages is based on the framework that they map to. The metapackages adopt the highest version number of the framework (for example, netstandard1.6) it maps to in its package closure. 
+Versioning for .NET Core metapackages is based on the .NET Core version they are a part of.
 
-The patch version for the metapackage is used to represent updates to the metapackage to reference updated packages. Patch versions will never include an updated framework version. As a result, the metapackages are not strictly SemVer compliant because their versioning scheme doesn't represent the degree of change in the underlying packages, but primarily the API level. 
+For instance, the metapackages in .NET Core 2.1.3 should all have 2.1 as their `MAJOR` and `MINOR` version numbers.
 
-There are two primary metapackages for .NET Core.
+The patch version for the metapackage is incremented every time any referenced package is updated. Patch versions will never include an updated framework version. As a result, the metapackages are not strictly SemVer compliant because their versioning scheme doesn't represent the degree of change in the underlying packages, but primarily the API level. 
 
-**NETStandard.Library**
-
-- v1.6 as of .NET Core 1.0 (these versions won't typically or intentionally match).
-- Maps to the `netstandard` framework. 
-- Describes the packages that are considered required for modern app development and that .NET platforms must implement to be considered a [.NET Standard](../../standard/net-standard.md) platform.
+There are currently two primary metapackages for .NET Core.
 
 **Microsoft.NETCore.App**
 
 - v1.0 as of .NET Core 1.0 (these versions will match).
 - Maps to the `netcoreapp` framework.
 - Describes the packages in the .NET Core distribution.
 
-Note: [`Microsoft.NETCore.Portable.Compatibility`](https://www.nuget.org/packages/Microsoft.NETCore.Portable.Compatibility) is another .NET Core metapackage. It doesn't map to a particular framework, so versions like a package.
+Note: [`Microsoft.NETCore.Portable.Compatibility`](https://www.nuget.org/packages/Microsoft.NETCore.Portable.Compatibility) is another .NET Core metapackage that exists to enable compatibility with pre .NET Standard implementation of .NET. It doesn't map to a particular framework, so versions like a package.
+
+**NETStandard.Library**
 
-### Frameworks
+`NETStandard.Library` describes the packages that are considered required for modern app development and that .NET platforms must implement to be considered a [.NET Standard](../../standard/library.md) platform.
```

Comment:
I don't like the term "modern app development" as used here. I think a lot of readers might think that .NET Standard does not apply to Console applications (are they 'modern'?).  Are web sites "apps"? I would just drop "for modern app development" and say "are considered required and that .NET platforms ...."